src/Compilers/Core/Portable/CodeGen/SequencePointList.cs

```diff
@@ -124,6 +124,35 @@ public ImmutableArray<Cci.SequencePoint> GetSequencePoints(DebugDocumentProvider
                 current = current._next;
             }
 
+            // Find the document for the first non-hidden sequence point (issue #4370)
+            current = this;
+            while (current != null)
+            {
+                for (var i = 0; i < this._points.Length; i++)
```

Comment:
No need to use "this.". I believe we've been moving away from that style since we started to use _ and s_ prefixes.